integration-tests/src/main/java/org/apache/polaris/service/it/env/Server.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.it.env;
+
+import java.net.URI;
+
+public interface Server extends AutoCloseable {
+  URI baseUri();
+
+  AuthToken adminToken();
+
+  ClientCredentials adminCredentials();
```

Comment:
Where has the `Snowman` user gone?

Reply:
Into the Nothing. The Server Manager provides "root" (admin) credentials. Tests create extra principals when they need to.

Reply:
I am fine with that, although my impression was that the snowman user was introduced precisely to avoid using the root user for all admin tasks. Let's wait and see what @andrew4699 says.

Reply:
I moved the work done by the `SnowmanCredentialsExtension` into specific test classes... hopefully maintaining the same functionality as before (the "snowman" principals used to be created using "admin" tokens too). Those can be found by searching for "snowman" in Principal names in the new test code. I believe explicit principal manipulation by test code is easier to follow and maintain than if it is done by a test extension. 

The principal and role manipulation logic moved to the methods in the new `ManagementApi` class.